### DISCUSSION TITLE
mspider/user-guide/cluster-management/join-clus

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3726] [Announcements]
# mspider/user-guide/cluster-management/join-clus

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/cluster-management/join-clus.html

**Comment by nicole-lihui** [comment]
接入检测需要更新
